gen/cheader.tmpl

```diff
@@ -131,15 +131,9 @@
  * For info on how this is used in various places, see \ref Strings.
  */
 typedef struct WGPUStringView {
-    char const * WGPU_NULLABLE data;
```

Comment:
note: I have this here because it's where we want it to be eventually: http://github.com/webgpu-native/webgpu-headers/issues/190
and since it's not autogenerated it won't get fixed along with everything else when we finally fix it.
But I guess it doesn't matter, once we do that we'll actually test it (set the macro and compile test) and presumably we'll find this again.